### ISSUE TITLE
Twilio Link Shortening 

### ISSUE THREAD

**Issue by HanaShiho** [episphere/connect#1000]
We want to incorporate link shortening in our text messages to monitor click tracking via Twilio. However, we have two limitations. First, the current setup does not allow the links to bypass the log-in page. Second, we lack a way to auto-navigate someone to a particular survey once they have signed in. Together, this means all links would take users to MyConnect log-in page. Despite this caveat, we can still gauge the gross number of clicks for each contact attempt and thus, which contact attempt results in the most clicks, both in terms of comparing email to SMS and 1st Contact vs. 2nd Contact, etc. 

@jhflorey here is the [Link Shortening Onboarding Guide](https://www.twilio.com/docs/messaging/features/link-shortening/onboarding-guide) for your reference.

**Comment by sonyekere** [comment]
Hi @depietrodeanna. Is this still an issue we’d like to address? If so, do you have a milestone in mind?

**Comment by depietrodeanna** [comment]
yes, this would be good to have but it is not the highest priority to get in. Some time over summer if the team could put eyes on the link shortening guide and dig into discussion with me/Ops, that would be good. I'll assign July as milestone but that is flexible. @sonyekere
